setup.py

```diff
@@ -47,6 +44,7 @@
     classifiers=[
         'Development Status :: 4 - Beta',
         'Intended Audience :: Developers',
+        'License :: OSI Approved :: MIT License',
```

Comment:
No license this short will preserve patent rights by those who use or derive works from the software. Most OSS licenses implicitly grant (and some explicitly permit) non-exclusive practice rights.¹ 

--

¹ This is not intended as legal advice. 😉